### PR TITLE
xx-astroimage

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

**Comment by Cadair** [comment]
@astrofrog take a quick look over this please?

**Comment by SolarDrew** [comment]
+1

**Comment by astrofrog** [comment]
When showing the header, why iterate over the keys/values, instead of just `print(hdulist[0].header)`?

**Comment by Cadair** [comment]
So that they were easier to read.

File: xx-astroimage/Astro_Images.ipynb [hunk 1]
```diff
@@ -0,0 +1,993 @@
...
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from sunpy.data.sample import AIA_171_IMAGE\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "hdulist = fits.open(AIA_171_IMAGE)\n",
+    "hdulist.verify('silentfix')\n",
+    "for k, v in hdulist[0].header.items():\n",
```
Comment (astrofrog) [inline xx-astroimage/Astro_Images.ipynb]: `print(hdulist[0].header)`

**Comment by astrofrog** [comment]
Just for info, the `wcsaxes` import isn't actually needed

**Comment by astrofrog** [comment]
Is the header `Combining WCS and wcsaxes` needed?

**Comment by astrofrog** [comment]
In the section `Overplotting in Pixel Coordinates` and `Overplotting in World Coordinates`, the overlay might be more obvious in e.g. orange.

**Comment by astrofrog** [comment]
Same comment about overlay color for the sun - maybe something light bright green would be good for that one.

**Comment by astrofrog** [comment]
Otherwise, :+1:

**Pull request merged**